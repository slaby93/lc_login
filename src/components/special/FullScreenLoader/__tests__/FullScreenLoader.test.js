import React from 'react';
import { shallow } from 'enzyme';
import FullScreenLoaderStyled, { FullScreenLoader } from './../FullScreenLoader';

describe('<FullScreenLoader />', () => {
  test('CSS', () => {
    const mountedComponent = shallow(<FullScreenLoaderStyled />);
    expect(mountedComponent).toMatchSnapshot();
  });

  test('HTML', () => {
    const mountedComponent = shallow(<FullScreenLoader
      className="test"
    />);
    expect(mountedComponent).toMatchSnapshot();
  });
});
