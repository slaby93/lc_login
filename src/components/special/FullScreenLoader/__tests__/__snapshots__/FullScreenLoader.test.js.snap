// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FullScreenLoader /> CSS 1`] = `
.c0 {
  position: fixed;
  width: 100vw;
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding-top: 20%;
  z-index: 1;
  top: 0px;
  left: 0px;
}

<FullScreenLoader
  className="c0"
/>
`;

exports[`<FullScreenLoader /> HTML 1`] = `
<div
  className="test"
>
  <div
    className="ball"
  />
  <div
    className="ball"
  />
  <div
    className="ball"
  />
  <div
    className="ball"
  />
  <div
    className="ball"
  />
  <div
    className="ball"
  />
  <div
    className="ball"
  />
</div>
`;
